roles/tomcat7/tasks/tomcat_service.yml

```diff
@@ -12,23 +12,10 @@
     group: "root"
     mode: 0755
 
-- name: Copy Tomcat7 Configs
-  become: yes
-  become_user: "root"
-  template:
-    src: "templates/var/lib/tomcat7/conf/{{ item }}.j2"
-    dest: "/var/lib/tomcat7/conf/{{ item }}"
-    owner: "root"
-    group: "{{ tomcat_group }}"
-    mode: 0644
-  with_items:
-    - "server.xml"
-    - "web.xml"
-
-- name: Start & Enable the Tomcat7 Service
+- name: Stop & Enable the Tomcat7 Service
   become: yes
   become_user: root
   service:
     name: tomcat7
-    state: started
-    enabled: true
\ No newline at end of file
+    state: stopped
```

Comment:
I think you won't have to make this change if `install_tomcat_service` was set to false when calling the role.